content/contribute/coding/add-secret-store.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Adding Secret Stores
```

Comment:
Not sure how to handle this, but I'd like to keep stuff relating to alpha or beta features very clearly delineated from everything else - under their own section, or some kind of very obvious banner/header?

Reply:
added in a unique PR. https://github.com/crossplane/docs/pull/263